reference/service/orgs.md

```diff
@@ -18,12 +18,53 @@ Adding a new Pulumi organization can be done directly from the Pulumi Cloud Cons
     <button class="button">ADD ORGANIZATION</button>
 </a>
 
-## GitHub-backed Organizations
+## GitHub and GitLab-backed Organizations
```

Comment:
Since you go into details about the different 3rd party organization types below (the table is super-useful), we can probably make the heading more generic. Perhaps the section heading could be "Imported Organizations", or "Importing from 3rd Party Services"?